src/Umbraco.Web/Security/Providers/MembersMembershipProvider.cs

```diff
@@ -7,13 +7,14 @@
 using Umbraco.Core.Models;
 using Umbraco.Core.Security;
 using Umbraco.Core.Services;
-using Umbraco.Core.Models.Membership;
 using Umbraco.Web.Composing;
 using System;
 using Umbraco.Net;
 
 namespace Umbraco.Web.Security.Providers
 {
+    //TODO: Delete: should not be used
+    [Obsolete("We are now using ASP.NET Core Identity instead of membership providers")]
```

Comment:
This needs to be deleted ultimately, once all has been replaced in terms of macros. Have obsoleted for now